specificatie/openapi.yaml

```diff
@@ -248,17 +248,19 @@ paths:
       summary: "vindt verblijfsobjecten, ligplaatsen, standplaatsen"
       description: >-
         Zoek actuele adresseerbare objecten (verblijfsobjecten, standplaatsen of ligplaatsen) met:
-        
+
         1. een nummeraanduidingidentificatie of
-        
+
         2. een pandidentificatie of
-         
-        3. een lijst met pandidentificaties.
 
+        3. een lijst met pandidentificaties of
+
+        4. een geometrische contour (rechthoek) in combinatie met status geconstateerd, oppervlakte, gebruiksdoel en/of type.
```

Comment:
@JohanBoer @strijm Waarom wordt met deze beschrijving het gebruik van bijv. gebruiksdoel uitgesloten op de andere parameters? Bijvoorbeeld in een pand (grote flat) zoeken naar alle verblijfsobjecten met gebruiksdoel "woonfunctie" (om serviceruimtes enz. eruit te filteren en alleen woningen te krijgen)?

Reply:
Ik vermoed dat dit gekomen is door hoe de user-stories #331,  #332 en #335 zijn geformuleerd. Daardoor staan in de goals-canvas deze parameters ook alleen in combinatie met de contour genoemd. 

Uiteraard kan dat aangepast worden indien gewenst (en zoals je aangeeft is daar een casus voor).